input/weapon-acquisition/event.json

```diff
@@ -63,7 +65,7 @@
     10099,10101,10108,10911,10940,10949,10962,
     // Festival: Halloween
     10121,10130,10132,
-    11053,11065,11069,
+    11055,11062,11071,
```

Comment:
Not clear to me these are dropped by an event. Seems they come from various trick or treat bags, or more reasonably purchased.